### ISSUE TITLE
JDK9 Support: Remove deleted GC flags

### ISSUE THREAD

**Issue by ash211** [palantir/gradle-java-distribution#226]
Compare these two links:

- "Options That Won’t Work " section of https://www.azul.com/jdk-9-pitfalls-for-the-unwary/
- args we set at https://github.com/palantir/sls-packaging/blob/2.4.0/gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.groovy#L34

We might need to remove:
- PrintGCDateStamps
- UseGCLogFileRotation
- GCLogFileSize
- NumberOfGCLogFiles

Possibly there are others as well, looking at JEP 214 on that page.

cc @schlosna

**Comment by schlosna** [comment]
Yeah, I expect that we need to allow selection of Java 8 or 9, and derive appropriate default flags for each. We'll likely need to enable the module escape valves for a bit.
